bftengine/tests/messages/NewViewMsg_test.cpp

```diff
@@ -32,15 +32,15 @@ TEST(NewViewMsg, base_methods) {
   EXPECT_EQ(msg.newView(), viewNum);
   EXPECT_EQ(msg.elementsCount(), 0);
 
-  for (uint8_t i = 1; i <= config.fVal * 2 + config.cVal * 2 + 1; ++i) {
+  for (uint8_t i = 1; i <= static_cast<uint8_t>(config.fVal * 2 + config.cVal * 2 + 1); ++i) {
```

Comment:
This is a super weird digest construction. I guess since it's only a test and we know the quorum size fits in 8 bits it's fine, but I'd still suggest changing how bytes are chosen rather than casting.

Reply:
In general, this test makes sure that any data stored in the message is retrievable.
The message itself does not alter or relate to the content of the digest.

Anyway, I agree that the casing looks ugly: fixed the casting and digest construction.